openstack/elektra/charts/postgres/templates/backup-configmap.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.backup.enabled }}
```

Comment:
There is no special reason. If you say it should be an ENV, we change it just in the backup container and delete the configmap.